### PR TITLE
run tests with options to prefix build root path

### PR DESCRIPTION
**Pull request by deepakrkris**

### CODE REVIEW

File: test/common/index.js [hunk 1]
```diff
@@ -75,13 +75,13 @@ exports.mustNotCall = function(msg) {
...
 
-exports.runTest = async function(test, buildType) {
+exports.runTest = async function(test, buildType, relativeBuildPath = process.env.REL_BUILD_PATH || '') {
   buildType = buildType || process.config.target_defaults.default_configuration || 'Release';
 
   const bindings = [
-    `../build/${buildType}/binding.node`,
-    `../build/${buildType}/binding_noexcept.node`,
-    `../build/${buildType}/binding_noexcept_maybe.node`,
+    `${relativeBuildPath}../build/${buildType}/binding.node`,
```
[the remark this answers is not in the document]
Comment (deepakrkris) [reply test/common/index.js]: @legendecas @mhdawson I have added path.join and renamed prefix to "buildPathRoot"

**Comment by deepakrkris** [comment]
@KevinEady @gabrielschulhof , this PR is ready to review

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
Landed as 62b666c34ce6b86117626948003a95931d9c2141

**Pull request closed without merging**